mapswipe_workers/sample_data/build_area/solid_waste_tutorial_screens.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
@laurentS this basically tells you for each screen the text strings. these are already in firebase as well. for simplicity it might be the best, if you change in firebase directly if needed.

Reply:
I think it would make it easier (at least on the app side) if the screens were 0-indexed, because with this format firebase returns an array of screens with the first element being `null`, which is a bit weird.

Reply:
Another tiny bug: xMax in the group was 117, when the tasks actually stop at 115. I've fixed in firebase, but you might want to check if there's a bug in the code that generated the data.

Reply:
We also need to think about how we're going to transition from the old tutorial format to the new one: the "old" app and the "new" one won't be able to use the same tutorial content, so there's probably going to be a period where either the old or the new version is broken for the tutorial. I can't think of a good way to handle this right now, unless maybe by creating a tutorial that has both the "categories" and "screens" in it, which we'd keep until the old version of the app is not around anymore (and we show an announcement asking users to update?)